library/math/numbersLine/src/components/ShowJumps.tsx

```diff
@@ -1,17 +1,23 @@
 import { TypesElement } from "@/type/elements";
 import Jump from "./Jump";
 import { useNumbersLineContext } from "@/context/numbersLineContext";
-import { useState } from "react";
+// import { useRef } from "react";
 
 const ShowJumps = () => {
-  const [idClick, setIdClick] = useState(-1);
-  const { dragElements } = useNumbersLineContext();
+  const { dragElements, setIdDraggElementClick } = useNumbersLineContext();
+  // const wrapperRef = useRef<HTMLDivElement>(null);
 
   return dragElements.map(
     (element) =>
       element.type == TypesElement.jump && (
```

Comment:
this query doesnt have to be before you call showJump component? and if it true then you have to rebder the component.